uktrade/data-hub-frontend
# Add form to archive a pipeline item

## Description of change

This is only available by changing the URL whilst be build more functionality. This allows a user to archive a pipeline item

## Test instructions

Add a company to your pipeline, click edit and repace `/edit` with `/archive` once a reason has been entered it should successfully `POST` to the API and then return you to the pipeline. You can then inspect the item on the pipeline to confirm `archive: true` is there for the item. We are working in functionality to hide the archived items in a subsequent PR.

## Screenshots

![archive_form_5](https://user-images.githubusercontent.com/1481883/84906258-95f45200-b0a9-11ea-8a5c-5bc3cb2863df.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
